### PR TITLE
#162725260 Users should be able to rate an article

### PR DESCRIPTION
**Pull request by calvinpete**
#### What does this PR do?
This PR adds functionality for articles to be rated by readers. And In case one fetches an article or all articles, he/she should be able to see the average ratings of the article.

##### Modify fetch articles endpoint to have 
```
{
      "average_rating": <average of the ratings scored>
}
```
##### Implement an endpoint
- Rate an article
`POST/api/articles/<slug>/rate`

#### How should this be manually tested?
- First things first, fetch this branch by running the command `git fetch origin ft-rate-article-162725260` in the terminal.
- Check into this branch by running `git checkout ft-rate-article-162725260`
- Activate your virtual environment by running `source venv/bin/activate`
- Drop your current database and create a new one by running `create database <database name>;` and `drop database <database name>;` in the psql shell
- Apply the new changes to the models by running `python manage.py migrate`
- Start the application by running `python manage.py runserver`
- Click the `Run in postman` button to test the endpoint as shown in the screenshots below

#### What are the relevant pivotal tracker stories?
[#162725260](https://www.pivotaltracker.com/story/show/162725260)

#### Rate an Article
image

#### Already Rated
image

#### Average Rating
image

#### POSTMAN
[Run in Postman](https://app.getpostman.com/run-collection/b7f9d4812b20fd57fe74)

### CODE REVIEW

File: authors/apps/articles/serializers.py [hunk 1]
```diff
@@ -86,3 +89,20 @@ def get_slug(self, obj):
 
     def get_description(self,obj):
        return obj.article.description
+
+
+class ArticleRatingSerializer(serializers.ModelSerializer):
+
+    title = serializers.SerializerMethodField()
+    author = serializers.SerializerMethodField()
+    rating = serializers.IntegerField(required=True, max_value=5, min_value=1)
```
Comment (ja-odur) [inline authors/apps/articles/serializers.py]: @calvinpete please set the min_value to zero

Comment (calvinpete) [reply authors/apps/articles/serializers.py]: Okay.

**Review by ja-odur — APPROVED** [review]
LGTM

**Review by mariamiah — APPROVED** [review]

**Review by armstrongsouljah — APPROVED** [review]
LGTM

**Pull request merged**